### PR TITLE
Add FAQ about already registered certificates

### PR DESCRIPTION
**Pull request by larswmh**
EN:
image
DE:
image

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@larswmh 

The new article looks fine!

Will you recommend getting certificates corrected? I know it's a hot topic!

**Comment by larswmh** [comment]
@MikeMcC399 Thanks! In the last commit, among other things, I added a comment at the end that prompts the reader to contact the issuer in case of an incorrect certificate

**Review by DerVogel2020 — CHANGES_REQUESTED** [review]

File: src/data/faq_de.json [hunk 1]
```diff
@@ -212,6 +212,18 @@
...
...kate auf verschiedenen Ebenen dargestellt und gruppiert: <ul><li><b>Reiter 'Zertifikate'</b>: Unter dem Reiter 'Zertifikate' können Nutzer*innen alle vorhandenen Zertifikate nach Personen gruppiert sehen. Überprüfen Sie, ob mehrere Personen sichtbar sind, indem Sie in dieser Ansicht nach unten scrollen. Häufig unterlaufen bei der Austellung der Zertifikate Fehler, beispielsweise durch das unregelmäßige Verwenden eines Zweitnamens, Titels oder einer unterschiedlichen Angabe des Geburtsdatums. Jegliche Abweichungen haben zur Folge, dass Zertifikate einer 'neuen Person' zugeordnet werden.</li><li><b>Alle Zertifikate einer Person</b>: Durch das Tippen auf die Karte einer Person können Sie alle Zertifikate sehen, die für diese Person in der App vorhanden sind. Überprüfen Sie, unter welcher Person das von Ihnen eingescannte Zertifikat zu finden ist.</li><li><b>Zertifikat-Details</b>: Durch die Auswahl eines Zertifikats können Sie Details zum jeweiligen Zertifikat einsehen. Überprüfen Sie den standardisierten Namen und das Geburtsdatum des Zertifikats. Diese Daten müssen für alle Zertifikate gleich sein, damit sie unter allen Zertifikaten einer Person korrekt angezeigt werden.</li></ul>",
```
[the remark this answers is not in the document]
Comment (larswmh) [reply src/data/faq_de.json]: Thanks, fixed in the latest commit

File: src/data/faq.json [hunk 2]
```diff
@@ -213,6 +213,27 @@
...
+                            "If you continue to receive the error message and have been able to verify that the certificate is indeed not already in your app, please reach out to us by mail at<a href='corona-warn-app.opensource...sap...com' class='email'>corona-warn-app.opensource...sap...com</a>. If your certificate has been issued incorrectly, please contact the issuer of the certificate."
```
Comment (MikeMcC399) [inline src/data/faq.json:234]: Missing space after "please reach out to us by mail at"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Missing space, noted inline.

File: src/data/faq.json (hunk 2 above)
Comment (larswmh) [reply src/data/faq.json]: Fixed in latest commit, thanks!

**Review by dsarkar — APPROVED** [review]

**Pull request merged**